custom_components/tuya_local/devices/techinc_smart_3m_neon_strip.yaml

```diff
@@ -0,0 +1,187 @@
+name: Tech Inc 3m neon strip
+products:
+  - id: szzsfdqcrqkczzfi
+    name: TI 3M neon light strip
+primary_entity:
+  entity: light
+  dps:
+    - id: 20
+      type: boolean
+      name: switch
+    - id: 21
+      type: string
+      name: color_mode
```

Comment:
Since there is only one real color mode here, I suggest using
```
name: effect
mapping:
  - dps_val: colour
    value: "off"
```
The rest can be the same.

Reply:
Changed